cocotb/binary.py

```diff
@@ -64,22 +64,22 @@ def _clog2(val):
         exp += 1
 
 
-class BinaryRepresentation():
-    UNSIGNED         = 0  # noqa
-    SIGNED_MAGNITUDE = 1  # noqa
-    TWOS_COMPLEMENT  = 2  # noqa
+class BinaryRepresentation():  # noqa
+    UNSIGNED         = 0  #: Unsigned format
+    SIGNED_MAGNITUDE = 1  #: Sign and magnitude format
+    TWOS_COMPLEMENT  = 2  #: Two's complement format
```

Comment:
What's with the addition of the colons?  I think it only exists on the first line because it's the beginning of a class, and that too comes before the comment.

Reply:
``#:`` is the markup for a special comment "string" for Sphinx.